doc/5_saving.md

```diff
@@ -471,6 +473,53 @@ val table = TableDef("test","words",Seq(p1Col),Seq(c1Col, c2Col),Seq(rCol))
 val rddOut = rdd.map(s => outData(s._1, s._2(0), s._2(1), s._3))
 rddOut.saveAsCassandraTableEx(table, SomeColumns("col1", "col2", "col3", "col4"))
 ```
+## Deleting rows and columns
+`RDD.deleteFromCassandra(keyspaceName, tableName)` delete rows from specified Cassandra table, using data from the RDD as primary keys.
+
+Keys RDD could be loaded from the same table:
```

Comment:
I have a pretty common Example syntax now in the docs which makes it so we can easily hyperlink specific examples.

See

> #### Example Saving an RDD of Tuples with Default Mapping
> Assume the following table definition:
> ```sql
> CREATE TABLE test.words (word text PRIMARY KEY, count int);
> ```
> 
> ```scala
> val collection = sc.parallelize(Seq(("cat", 30), ("fox", 40)))
> collection.saveToCassandra("test", "words", SomeColumns("word", "count"))
> ```
>     
>     cqlsh:test> select * from words;

So we should add in the 'scala' tag for the markdown code to get colors right. 

It also would probably help to add in a real example here 

Deleting partitions from Words which have a count less than X or something.